Character set and collation updates (#19976)
pingcap/docs: sql-statements/sql-statement-show-collation.md

## Examples

<CustomContent platform="tidb">

> When [the new collation framework](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) is enabled (the default) the output looks like this.

Reviewer: ```suggestion
When [the new collation framework](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) is enabled (the default), the example output is as follows:
```
